mkosi/bootloader.py

```diff
@@ -585,6 +585,10 @@ def find_and_install_shim_binary(
 
                 rel = p.relative_to(context.root)
                 if (context.root / output).is_dir():
+                    # The ESP wants .efi files, not .efi.signed or .efi.signed.latest
+                    if rel.suffix and rel.suffix != ".efi":
+                        left_stem, _ = rel.name.split(".", maxsplit=1)
+                        rel = rel.with_name(f"{left_stem}.efi")
```

Comment:
Modifying `rel` messes up the log message. Please only modify `output` and not `rel`